### ISSUE TITLE
No option to upload multiple media items

### ISSUE THREAD

**Issue by dartiss** [WordPress/gutenberg#2738]
## Issue Overview
With Guternberg you are no longer able to upload multiple media items at once.

## Expected Behavior
With the current MCE editor, when selecting the 'Add Media' button you can then select multiple media items to upload at once,

## Current Behavior
You can only upload media now when choosing to insert it and then only once file.

**Comment by youknowriad** [comment]
if you select to insert a Gallery block, you have the possibility to upload multiple media items because you can use multiple items while when inserting an image you can only upload and use one.

Does this answer your question?

**Comment by dartiss** [comment]
That's one solution, yes. Let me explain how I often work... I start writing a post and then, at some point, want to bulk upload images to then, when needed, insert into the post. I do them all at one time as I have a plugin installed that compresses the images and that makes them slower to upload than normal.

In the past I would have simply used the add media option to upload them easily in one go and, it seems to me, a part of the existing process is missing as a result. What I don't know is whether I'm unusual in the way I do this.

**Comment by youknowriad** [comment]
I guess this is more a use-case of the media library and in these situations, you'll probably want to go the Media page and bulk upload there. I'm not trying to dismiss your use-case, maybe at some point we'll have to bring a global add media button to Gutenberg for convenience, but we're more interested in thinking the best UX using blocks right now.

Also related, you can drag and drop files into the editor, for now, it works only for a single image, but we'll be adding the possibility to drag and drop multiple images at once soon.

**Comment by karmatosed** [comment]
I feel right now this is out of scope for Gutenberg and verges into media library. That's not to say that we won't work on this in later versions. For now, lets close, thanks.

**Comment by badfeather** [comment]
Though this is closed, I disagree that this is media library territory. Uploading images via the post editor attaches the media to the post, which is useful for locating images quickly, as well as retrieving post attachments via template functions. I am also used to the workflow that @dartiss uses, where I typically upload all the media at once, then insert images while I'm fleshing out the content of the post. While bulk uploading via the gallery block works for this, technically, it's a bit of a hacky solution.
